src/Microsoft.AspNetCore.Antiforgery/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -22,7 +21,8 @@ public class DefaultAntiforgeryTokenGenerator : IAntiforgeryTokenGenerator
             _additionalDataProvider = additionalDataProvider;
         }
 
-        public AntiforgeryToken GenerateCookieToken()
+        /// <inheritdoc />
+        public AntiforgeryToken GenerateCookieToken(HttpContext httpContext)
```

Comment:
Any implementation other than ours would likely need it. Our default implementation just generates a 'dump' token. If you wanted to do something based on the user's claims/identity you'd need the context.

Basically it seemed very short-sighted and limiting that the methods operating on the request token have access to context, and the methods operating on the cookie token do not.